components/create_comment/create_comment.jsx

```diff
@@ -281,6 +286,11 @@ export default class CreateComment extends React.PureComponent {
             return;
         }
 
+        if (this.props.rootDeleted) {
+            this.showPostDeletedModal();
```

Comment:
Now that this is driven purely by a prop that itself won't change, will a re-render pass trigger the modal to spontaneously reappear after being dismissed? Or from another angle, if I try to post again after dismissing, do I still get the modal?